manifests/stable/runoncedurationoverride.crd.yaml

```diff
@@ -1,36 +1,399 @@
+---
```

Comment:
Is this CRD auto generated or by some tool?. Because I manually created it and I wonder I'm doing something wrong

Reply:
I ended up with `make regen-crd`. Though, it's still partially manual. There are always issues with generating a CRD :-|